ocs_ci/deployment/vmware.py

```diff
@@ -1386,7 +1386,13 @@ def destroy_cluster(self, log_level="DEBUG"):
                 rbd_name = config.ENV_DATA.get("rbd_name") or defaults.RBD_NAME
                 # get external cluster details
                 host, user, password, ssh_key = get_external_cluster_client()
-                external_cluster = ExternalCluster(host, user, password, ssh_key)
+                external_cluster = ExternalCluster(
+                    host,
+                    user,
+                    password,
+                    ssh_key,
+                    jump_host=config.EXTERNAL_MODE.get("ssh_jump_host"),
+                )
```

Comment:
why do we need this change for vmware and ocs_upgrade ?

Reply:
I've applied it generally to all the places where we are using `ExternalCluster` class (to have this option for any environment).